sdk/appconfiguration/app-configuration/src/policies/throttlingRetryPolicy.ts

```diff
@@ -35,27 +41,51 @@ const StandardAbortMessage = "The operation was aborted.";
  * @internal
  */
 export class ThrottlingRetryPolicy extends BaseRequestPolicy {
-  constructor(nextPolicy: RequestPolicy, options: RequestPolicyOptions) {
+  private numberOfRetries = 0;
+  constructor(
+    nextPolicy: RequestPolicy,
+    options: RequestPolicyOptions,
+    private retryOptions: Pick<RetryOptions, "maxRetries" | "maxRetryDelayInMs"> = {}
```

Comment:
update the parameter type to use the local one?

Reply:
updated!